packages/js/src/utils/requester.ts

```diff
@@ -32,20 +32,20 @@ const getLogtoErrorParametersByResponse = async (
 
 export const createRequester = (fetchFunction?: typeof fetch) => {
   if (!fetchFunction && isNode()) {
-    throw new LogtoError('requester.not_provide_fetch');
+    throw new LogtoError({ code: 'requester.not_provide_fetch' });
   }
 
   return async <T>(...args: Parameters<typeof fetch>): Promise<T> => {
     const response = await (fetchFunction ?? fetch)(...args);
 
     if (!response.ok) {
       const logtoErrorMessage = await getLogtoErrorParametersByResponse(response);
```

Comment:
`const {error, error_description: errorDescription, message } = await getLogtoErrorParametersByResponse()`

Reply:
Improved, PTAL.

Reply:
Outdated, follow the new implementation.